### PR TITLE
added emails extractor

### PR DESCRIPTION
**Pull request by harshareddy794**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #60 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [ ] I have added my work in respective language folder.
* [ ] My changes generate no new warnings.

### CODE REVIEW

File: Python/Emails_extractor_from_web_pages/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Introduction
+Emails_extractor is a python script wihih eill extract all email in a web page using regular expressions
```
Comment (tejan-singh) [inline Python/Emails_extractor_from_web_pages/README.md]: please correct spellings in readme line no 2

File: Python/Emails_extractor_from_web_pages/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+urllib
+xlwt
+bs4
+datetime
+re
```
Comment (tejan-singh) [inline Python/Emails_extractor_from_web_pages/requirements.txt]: please specify this module name in line 5

**Review by tejan-singh — CHANGES_REQUESTED** [review]
please make changes as requested

File: Python/Emails_extractor_from_web_pages/requirements.txt (hunk 2 above)
Comment (harshareddy794) [reply Python/Emails_extractor_from_web_pages/requirements.txt]: That's re (Regular expressions)

**Comment by harshareddy794** [comment]
Done @tejan-singh 
Made all changes

File: Python/Emails_extractor_from_web_pages/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+urllib
```
Comment (tejan-singh) [inline Python/Emails_extractor_from_web_pages/requirements.txt]: cpd
cp

please specify requirements properly.

**Review by tejan-singh — CHANGES_REQUESTED** [review]
cp

please specify all requirements in readme.

**Comment by harshareddy794** [comment]
Made changes accordingly @tejan-singh

**Comment by tejan-singh** [comment]
Which operating system are you testing this script on? Windows or Linux

**Comment by harshareddy794** [comment]
> Which operating system are you testing this script on? Windows or Linux

I'm using windows

**Comment by tejan-singh** [comment]
Okey, I will talk to other mentors for review.

**Pull request merged**